cli/PrintToScreen.hpp

```diff
@@ -43,14 +43,13 @@ class PrintToScreen {
   static void PrintRelation(const CatalogRelation &relation,
                             StorageManager *storage_manager,
                             FILE *out);
-
+  static void printHBar(const std::vector<int> &column_widths,
+                        FILE *out);
```

Comment:
Please add one empty line before and after this function `printHBar`, respectively. Thanks!

Also, we need to modify this file as global functions within namespaces in a separate PR.